src/tests/ov_helpers/ov_snippets_models/include/subgraph_mha.hpp

```diff
@@ -45,7 +45,6 @@ class MHAFunction : public SnippetsFunctionBase {
 public:
     explicit MHAFunction(const std::vector<PartialShape>& inputShapes, const std::vector<ov::element::Type>& precisions, bool with_mul = true)
         : SnippetsFunctionBase(inputShapes), with_mul(with_mul), precisions(precisions) {
-        OPENVINO_ASSERT(input_shapes.size() == 4, "Got invalid number of input shapes");
         OPENVINO_ASSERT(precisions.size() == 4, "Got invalid number of input precisions");
```

Comment:
Were these changes made intentionally? If yes, could you please explain why?
It seems strange that input_shapes size could be not equal to 4, but precisions size still must be equal to 4